docs/modules/mobile/reconnection-logic.md

```diff
@@ -0,0 +1,74 @@
+<%(TOC)%>
+
+# Reconnection logic
+
+Mobile applications present new challenges in terms of handling client connections. A client may disconnect due to unstable connections or when pausing the application. We want to dive a bit deeper into the causes of a disconnect and how you can handle it in your game.
```

Comment:
```suggestion
Mobile applications present new challenges in terms of handling client connections. A client may disconnect due to unstable connections or when the application is put into a background state by the OS. We want to dive a bit deeper into the causes of a disconnect and how you can handle it in your game.
```

Reply:
To capture the specific behaviour of mobile devices vs just application is paused (which feels like a user-level action rather than a system one)